src/benchmarks/micro/corefx/System.IO.Compression/Brotli.cs

```diff
@@ -20,8 +20,8 @@ public Span<byte> Compress_WithState()
         {
             using (BrotliEncoder encoder = new BrotliEncoder(GetQuality(level), Window))
             {
-                Span<byte> output = new Span<byte>(CompressedFile.UncompressedData);
-                ReadOnlySpan<byte> input = CompressedFile.CompressedData;
+                Span<byte> output = new Span<byte>(CompressedFile.CompressedData);
```

Comment:
Shouldn't we be writing to an empty span rather than one that's already full of the correct data?  Maybe it doesn't matter since this is only measuring perf, but it seemed odd to me.

Reply:
We need a span that is big enough to write the entire output to it. 

`CompressedData` has a perfect size because it has already been compressed once in the benchmark argument setup: 

https://github.com/dotnet/performance/blob/c7ad6f2360a2fbf4d1c41b4d30159996bad8dc20/src/benchmarks/micro/corefx/System.IO.Compression/CompressedFile.cs#L30

However, as this PR has shown reusing fields in such a tricky way can lead to bugs. So most probably having `CompressionInput` and `CompressionOutput` fields would be better and more clear.